package.json

```diff
@@ -36,7 +36,8 @@
     "tailwindcss": "1.7.3",
     "v-tooltip": "2.0.3",
     "vue": "2.6.12",
-    "vue-hoc": "0.4.7"
+    "vue-hoc": "0.4.7",
+    "clipboard-copy": "3.1.0"
```

Comment:
This should be ordered alphabetically, I think you need to just run npm install, because npm re-orders dependencies in package.json. It might also update the reference in package-lock.json.

Reply:
npm install updated package-lock, order was done manually.